test/forms/CancelVATRegistrationFormSpec.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+class CancelVATRegistrationFormSpec extends FormBehaviours {
+  val form = CancelVATRegistrationForm.cancelVATRegistrationForm()
+  "CancelVATRegistrationForm" should {
+    "bind when an option is selected" in {
```

Comment:
If you make the changes to the form, could you add some more tests?

You can do `behave like mandatoryField(xxxx)` see https://github.com/hmrc/penalties-appeals-frontend/blob/aad967ac69169c691a862a56eb39f0076fbc3df1/test/forms/HasCrimeBeenReportedFormSpec.scala#L24

Reply:
Added it.